home.html

```diff
@@ -0,0 +1,97 @@
+                <h1 class="navbar-m-0">LUCENT QUIZ</h1>
+            </a>
+            <a class="navbar-m-left-auto navbar" href="#">
+                <h5>LOGIN</h5>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-github"></i>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-linkedin"></i>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+        </div>
+    </header>
+    <!-- Header Ends -->
+
+    <!-- Main Section Starts -->
+    <main class="main">
+        <!-- Hero Section Starts -->
+        <section id="hero" class="hero">
+            <div class="hero-wrapper">
+                <div class="hero-text-container">
+                    <h1>Let's Explore Quiz</h1>
+                    <P>
+                        What to improve your knowledge of the yogic principles? Then this is the right place to explore.
+                    </P>
+                    <a href="#category">
+                        <button class="btn btn-primary">Explore Now</button>
+                    </a>
+                </div>
+                <div class="hero-img-container">
+                    <img class="image-resp" src="./images/quiz.png" alt="hero image">
+                </div>
+            </div>
+        </section>
+        <!-- Hero Section Ends -->
+
+        <!-- Category Section Starts -->
+        <section id="category" class="category-section">
+            <h2>Categories</h2>
+            <div class="category-container">
+                <a class="card category card-box-shadow" href="./quiz/category/category.html">
+                    <img class="image-resp" src="./images/quiz.png" alt="image">
+                    <h4>Category Details</h4>
+                </a>
+                <a class="card category card-box-shadow" href="./quiz/category/category.html">
+                    <img class="image-resp" src="./images/quiz.png" alt="image">
+                    <h4>Category Details</h4>
+                </a>
+                <a class="card category card-box-shadow" href="./quiz/category/category.html">
+                    <img class="image-resp" src="./images/quiz.png" alt="image">
+                    <h4>Category Details</h4>
+                </a>
+            </div>
+        </section>
+        <!-- Category Section Ends -->
+
+    </main>
+    <!-- Main Section Ends -->
+
+    <!-- Footer Section Starts -->
+    <footer id="footer" class="footer">
+        <p>Made by Yaswanth Myneni</p>
+        <div>
+            <a href="#">
+                <i class="fa-brands fa-github social-links"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-linkedin social-links"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-twitter social-links"></i>
+            </a>
```

Comment:
There is a small blue colored dash next to the twitter and linkedin logos in the footer.